cloud_governance/policy/ec2_idle.py

```diff
@@ -0,0 +1,132 @@
+from datetime import datetime, timedelta
+
+from cloud_governance.common.aws.cloudwatch.cloudwatch_operations import CloudWatchOperations
+from cloud_governance.common.logger.init_logger import logger
+from cloud_governance.zombie_non_cluster.run_zombie_non_cluster_policies import NonClusterZombiePolicy
+
+
+class EC2Idle(NonClusterZombiePolicy):
+
```

Comment:
Add documentation for this policy 
when it trigger mail alert and when it stop ec2